### ISSUE TITLE
How to install manually?

### ISSUE THREAD

**Issue by jazzi** [CDrummond/lms-material#407]
First of all, thanks for such great work. Sadly here we have problem to connect to https://raw.githubusercontent.com, however it's ok to download the latest release as ZIP file, so is there possible to unzip the file and put it into the specific skin folder of Logitech Media Sever?

Ever tried to put unzip file into /usr/share/squeezeboxserver/Plugins/ and change the owner to squeezeboxsever as well as chmod 777 to all, after that "Material skin" is visible in the Plugin tap but shows "Failed to load", once choose Material as the interface, everything is gone, completely blank white. So I think this might because of my wrong handling.

* My LMS version is 8.2.0 (should have Material installed default but can not see it.)
* OS: Armbian, a debian ARM

Any tips will be appreciated, thanks.
jazzi

**Comment by CDrummond** [comment]
Create a `MaterialSkin` folder in ` /usr/share/squeezeboxserver/Plugins/` and unzip contents of (e.g.) [https://github.com/CDrummond/lms-material/releases/download/2.1.3/lms-material-2.1.3.zip](https://github.com/CDrummond/lms-material/releases/download/2.1.3/lms-material-2.1.3.zip) into that folder.

**Comment by jazzi** [comment]
After restart LMS, in the plugin tap, still showed error message "failed to load" while the home page becomes completely white blank, 
![screenshot-failed-materialskin](https://user-images.githubusercontent.com/1588914/105348243-01476900-5c23-11eb-81d7-d990e5eee0b3.png)
and see nothing from the log file.

**Comment by jazzi** [comment]
Problem resolved now, here is how I did it.

The installation of LMS failed to create /usr/share/squeezeboxserver thus all files that should exist within this directory missed, so I manually created this directory (/usr/share/squeezeboxserver), change owner and group to squeezeboxserver and nogroup, then uninstall LMS and reinstall it.

After everything is ready, unzip lms-material into /usr/share/squeezeboxserver/Plugins/MaterialSkin and change owner:group into squeezeboxserver:nogroup.

Final step is to restart LMS `sudo systemctl restart logitechmediaserver.service` and check your webUI again, wow Material Skin is there and working.

My OS is Armbian on zeroPi, don't know if other architect package can successfully create /usr/share/squeezeboxserver
